woocommerce/Settings_API/Abstract_Settings.php

```diff
@@ -35,6 +37,249 @@
  */
 abstract class Abstract_Settings {
 
+
+	/** @var string settings ID */
+	public $id = '';
```

Comment:
Do we need to initialize it here, since it will be in the constructor? 🤔

Reply:
I think we don't. Fixed in 05d04f0.